java/opendcs/src/main/java/decodes/tsdb/comprungui/CompRunGuiFrame.java

```diff
@@ -1135,6 +1194,7 @@ public List<CTimeSeries> doInBackground()
 							{
 								runme.addTimeSeries(ts);
 								inputs.add(ts);
+								myinputs.add(ts);
```

Comment:
I'm having a difficult time understanding why we need a 2nd array of the same inputs.